packages/docs-site/src/library/pages/components/index.md

```diff
@@ -10,7 +10,7 @@ header: true
 
 # Overview
 
-Components are reusable User Interface (UI) elements like buttons, text inputs or headers. They are a modular foundation for designing and developing applications and services. Their design is based on defined NELSON [styles](/styles).
+Components are reusable User Interface (UI) elements like buttons, text inputs or headers. They are a modular foundation for designing, developing applications and services. Their design is based on defined NELSON [styles](/styles).
```

Comment:
I am not sure that this reads as well without the "and".

Reply:
Thought the same, although it was copied directly from the Google Doc. Will clarify.